Assets/HoloToolkit/Input/Scripts/GazeManager.cs

```diff
@@ -11,34 +11,30 @@ namespace HoloToolkit.Unity
     public partial class GazeManager : Singleton<GazeManager>
     {
         /// <summary>
-        /// Maximum gaze distance, in meters, for calculating a hit from a GameOjects Collider.
+        /// Occurs when gaze enters a GameObjects collider.
```

Comment:
I was probably going to close the PR and open a new one to clean up some of this for easier review.  This was mainly for starting the discussion and letting people comment of the proposed changes.